docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx

```diff
@@ -1,18 +1,43 @@
 ---
 title: Enterprise License File
-description: Teleport enterprise license file configuration parameters and requirements
+description: How to manage your Teleport Enterprise license file.
 ---
 
-This guide explains the use of a license file with Teleport Enterprise.
+Self-hosted Teleport Enterprise subscriptions require a valid license. In this
+guide, we will show you how to manage your Teleport Enterprise license file.
 
-Commercial self-hosted Teleport subscriptions require a valid license. The
-license file can be downloaded from the [Teleport Customer
-Portal](https://dashboard.gravitational.com/web/login). When downloading the
-file, the products you are licensed to use will be listed in the browser.
+(!docs/pages/includes/cloud/call-to-action.mdx!)
+
+## Managing your license file
+
+In a self-hosted Teleport Enterprise cluster, the Teleport Auth Service reads a
+license file in order to determine the scope and validity of your subscription.
+Follow the steps below to give your Teleport Auth Service instances access to
+your license file.
+
+### Download your license file
+
+To obtain your license file, visit the [Teleport customer
+dashboard](https://dashboard.gravitational.com/web/login) and log in. Click
```

Comment:
@russjones do we have a name for the customer portal?

Reply:
Looks like "customer portal" is the most common usage, so I'll change it to that:

```
% grep -RohE "\[[A-Za-z ]+\]\(https://dashboard.gravitational.com/web/login\)" | sort | uniq -c | sort -n
   1 [Dashboard](https://dashboard.gravitational.com/web/login)
   1 [customer dashboard](https://dashboard.gravitational.com/web/login)
   3 [Customer Portal](https://dashboard.gravitational.com/web/login)
  10 [customer portal](https://dashboard.gravitational.com/web/login)
```